src/main/resources/swagger.api/folio-modules/inventory/inventoryInstance.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Inventory instance event data model",
+  "javaType": "org.folio.rest.jaxrs.model.InventoryInstance",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "id": {
+      "description": "UUID",
+      "$ref": "../common/uuid.json"
+    },
+    "staffSuppress": {
+      "type": "boolean"
+    },
+    "discoverySuppress": {
+      "type": "boolean"
+    }
```

Comment:
There is only id and suppress flags. Meaning that if we send such entity to be indexed in the search module, it could erase all previously saved data (if update is not incremental there). Have we checked if it's the case?

Reply:
Fixed.